worker/provisioner/lxd-broker.go

```diff
@@ -55,7 +55,11 @@ func (broker *lxdBroker) StartInstance(args environs.StartInstanceParams) (*envi
 	machineId := args.InstanceConfig.MachineId
 	bridgeDevice := broker.agentConfig.Value(agent.LxcBridge)
```

Comment:
I'm pretty sure this means it is looking at a config variable named "lxc-bridge" which is probably not what we want. Sorry to tack this onto your existing proposal, do you have a chance to double check this and move it to "lxd-bridge" ?